rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -504,6 +513,91 @@ def test_place_xdai_order(gnosis_inquirer, gnosis_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x2ef3e17313340294f80b2ac03f6e6d602da2f6617fdf9bb0aeda9d29f6a4960e')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=binance_sc_inquirer, tx_hash=tx_hash)  # noqa: E501
+    expected_events = [
+        EvmSwapEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=(timestamp := TimestampMS(1759334123000)),
+            location=Location.BINANCE_SC,
+            event_subtype=HistoryEventSubType.SPEND,
+            asset=A_BSC_BNB,
+            amount=FVal(raw_amount := '0.000988114325798632'),
+            location_label=binance_sc_accounts[0],
+            notes=f'Swap {raw_amount} BNB in a cowswap market order',
+            counterparty=CPT_COWSWAP,
+            address=GPV2_SETTLEMENT_ADDRESS,
+        ), EvmSwapEvent(
+            tx_hash=tx_hash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.BINANCE_SC,
+            event_subtype=HistoryEventSubType.RECEIVE,
+            asset=Asset('eip155:56/erc20:0xfb6115445Bff7b52FeB98650C87f44907E58f802'),
+            amount=FVal(received := '0.003503182735898775'),
+            location_label=binance_sc_accounts[0],
+            notes=f'Receive {received} AAVE as the result of a cowswap market order',
+            counterparty=CPT_COWSWAP,
+            address=GPV2_SETTLEMENT_ADDRESS,
+        ), EvmSwapEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.BINANCE_SC,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_BSC_BNB,
+            amount=FVal(fee_amount := '0.000011885674201368'),
+            location_label=binance_sc_accounts[0],
+            notes=f'Spend {fee_amount} BNB as a cowswap fee',
+            counterparty=CPT_COWSWAP,
+            address=GPV2_SETTLEMENT_ADDRESS,
+        ),
+    ]
+    assert expected_events == events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('binance_sc_manager_connect_at_start', BSC_NODES_TO_CONNECT)
+@pytest.mark.parametrize('binance_sc_accounts', [['0xc37b40ABdB939635068d3c5f13E7faF686F03B65']])
+def test_bnb_create_order(binance_sc_inquirer, binance_sc_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xd1bd1b511948bcbef89304a4a6004eed3e99bb36b90b19a814ab3b3719cc98ad')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=binance_sc_inquirer, tx_hash=tx_hash)  # noqa: E501
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=TimestampMS(1759334099000),
+            location=Location.BINANCE_SC,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_BSC_BNB,
+            amount=FVal('0.000056252'),
+            location_label=binance_sc_accounts[0],
+            notes='Burn 0.000056252 BNB for gas',
```

Comment:
```suggestion
            notes=f'Burn {gas_amount} BNB for gas',
```

the event below also.